identity/src/main/java/com/stripe/android/identity/navigation/DocSelectionFragment.kt

```diff
@@ -41,49 +44,20 @@ internal class DocSelectionFragment(
         identityViewModelFactory
     }
 
-    private lateinit var binding: DocSelectionFragmentBinding
```

Comment:
Can the corresponding XML file be deleted, too?

Reply:
nice catch, Done, also removed `consent_fragment.xml`